src/pages/Questionnaire.tsx

```diff
@@ -145,6 +146,9 @@ const calculateTotalScore = (questions: Question[], responses: Responses) => {
     totalScore = 100;
   }
 
+  // Round the total score to the nearest integer and cast it to a number
+  totalScore = round(totalScore, 0) as number;
```

Comment:
you shouldn't have to type cast it to `number`. TypeScript will automatically infer that `totalScore` is a number.
And just realizing that the logic from line 145 to 147 is not needed if you are rounding the `totalScore`